### PR TITLE
add two js quiz questions (promise, logical operator)

### PR DESCRIPTION
**Pull request by Starboy-Sharma**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -5810,5 +5810,23 @@ const fullQuiz = [
...
+    Explanation: "Value of x is 1 because undefined is a falsy value that's why 1 is assigned to the variable x",
+    Link: "https://www.freecodecamp.org/news/javascript-logical-operators/",
+  },
+  {
+    Question: "How to create promise inside a function in Javascript ?",
+    Answer: "return new Promise((resolve, reject))",
+    Distractor1: "return new Promise(resolve, reject)",
+    Distractor2: "return new Promise((resolve, reject))",
+    Distractor3: "return Promise((reject, resolve))",
+    Explanation: "You can use promise inside a function by return new Promise((resolve, reject) => {})",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:5828]: Proposed fix:
You can use a promise inside a function with the following syntax: return new Promise((resolve, reject) => {})

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @Starboy-Sharma ! 

Thank you for your quiz questions.
Please refer to my comments below to make those changes. 👍

**Comment by Starboy-Sharma** [comment]
Hi @jdwilkin4 
Thank you for your feedback. 
Changes you mention are now updated.

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -5813,19 +5813,19 @@ const fullQuiz = [
   {
     Question: "What is the value of x ? let x = undefined || 1;",
     Answer: "1",
-    Distractor1: "1",
-    Distractor2: "undefined",
+    Distractor1: "2",
+    Distractor2: "1",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:5817]: None of the distractors should have a correct answer.
Only the answer field should have the correct answer.

Please change back to Distractor2: "undefined",

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
If you could change that one distractor field to be an incorrect answer, then we can merge your questions.

File: src/data/full-quiz.js (hunk 2 above)
Comment (Starboy-Sharma) [reply src/data/full-quiz.js]: Okay, I got it. Thank you 👍

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
Congrats on your first PR for the Developer Quiz site.
We look forward to future contributions! 👍

**Comment by Starboy-Sharma** [comment]
Thank you so much @jdwilkin4. This is my first contribution to an open-source project. 🎉

**Pull request merged**